packages/admin/cms-admin/src/dam/DataGrid/FolderDataGrid.tsx

```diff
@@ -372,6 +376,26 @@ const FolderDataGrid = ({
             hideSortIcons: true,
             disableColumnMenu: true,
         },
+        {
+            field: "importSourceType",
+            headerName: intl.formatMessage({
+                id: "comet.dam.file.importSourceType",
+                defaultMessage: "Source",
+            }),
+            renderCell: ({ row }) => {
+                if (row.__typename === "DamFile") {
+                    if (row.importSourceType) {
+                        return importSourceTypeLabels?.[row.importSourceType];
+                    } else {
+                        return <FormattedMessage id="comet.dam.file.importSourceType.cmsAsset" defaultMessage="CMS Asset" />;
```

Comment:
all are CMS assets, no? (I would leave this blank)

Reply:
Discussed internally and removed fallback: 18666ab243671623993e16e1dc0bb2131e90ebfd